src/pages/map.tsx

```diff
@@ -137,25 +94,54 @@ const Index = ({ schedule }) => {
           height: '100vh',
           width: '100%'
         }}
+        onDrag={e => setActiveLoc(null)}
         onStyleLoad={loadedMap => {
           setMap(loadedMap);
           loadedMap.setCenter({ lat: -6.163088, lng: 106.836715 });
```

Comment:
maybe we can use this props instead `center={[106.836715, -6.163088]}`

Reply:
this will be triggered when rerender

Reply:
I'm sorry, wrong referencing the line.
I mean at line 100, 
So, on first render the map show Jakarta instead of other country. 

Possibly related to #12

Reply:
hi, @svspicious for that part.it will be fixed on my pr here #30